Documentation/ceph-object-multisite.md

```diff
@@ -23,18 +19,18 @@ This allows the ceph-object-store to replicate its data over multiple Ceph clust
 
 To review core multisite concepts please read the [ceph-multisite design overview](/design/ceph/object/ceph-multisite-overview.md).
 
-## Prerequisites
+# Prerequisites
 
 This guide assumes a Rook cluster as explained in the [Quickstart](ceph-quickstart.md).
 
 # Creating Object Multisite
 
 If an admin wants to set up multisite on a Rook Ceph cluster, the admin should create:
 
-    - A [realm](/Documentation/ceph-object-multisite-crd.md)
-    - A [zonegroup](/Documentation/ceph-object-multisite-crd.md)
-    - A [zone](/Documentation/ceph-object-multisite-crd.md)
-    - An [object-store](/Documentation/ceph-object-store.md) with the zone section
+1. A [realm](/Documentation/ceph-object-multisite-crd.md)
+2. A [zonegroup](/Documentation/ceph-object-multisite-crd.md)
+3. A [zone](/Documentation/ceph-object-multisite-crd.md)
+4. An [object-store](/Documentation/ceph-object-store.md) with the zone section
```

Comment:
With a markdown numbered list, just number them all as "1.", and the rendering will increment them.